repos/system_upgrade/common/actors/checkcustommodifications/libraries/checkcustommodifications.py

```diff
@@ -0,0 +1,38 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import CustomModifications
+
+
+def _create_report(msgs, report_type, component=None, hint=None):
+    filtered_msgs = [m for m in msgs or [] if m.type == report_type and (not component or m.component == component)]
+    if not filtered_msgs:
+        # Nothing to report
+        return
+    discovered_files = '\n'.join(
+            ['- {filename}{actor}'.format(filename=m.filename,
```

Comment:
Can you put the `\n-` "separator" in a global variable called `FMT_LIST_SEPARATOR`?  We do this everywhere until we have some standard format for these lists. You can easily grep for an example of this.